ocrd_tool.md

```diff
@@ -10,6 +10,84 @@ services](swagger).
 
 To validate a `ocrd-tool.json` file, use `ocrd ocrd-tool /path/to/ocrd-tool.json validate`.
 
+## Standard parameters
+
+There is a number of parameters common to all processors that MUST be supported by processors.
```

Comment:
I thought the idea was actually to provide extra parameters __automatically__ to all processors (without any need for specification/validation and implementation in the individual processors). So this would have to enter the spec at CLI.md, not here – right?

And besides using extra parameters, I have [proposed](https://github.com/OCR-D/core/issues/376#issuecomment-562349028) other mechanisms (like extra CLI options, and environment variables), too. Shouldn't we at least discuss these options first?

Reply:
Hmm, the point here was to have a place for that discussion. I wrote down what came to mind at the place where it fitted best. I wouldn't want to necessarily force everyone to implement a `dpi` parameter they do not need (core might help here though). Could be overrideable via env var, getopt-style extension etc. We just need a solution soon and a PR is more urgent than issues.

> So this would have to enter the spec at CLI.md, not here – right?

Again, I'm open for changing the details, as long as there is one place with a fragid `#standard-parameters` that describes these "special" "parameters"/"flags"/"settings".